instrumentation/classloaders/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/javaclassloader/ClassLoaderInstrumentationModule.java

```diff
@@ -20,7 +20,10 @@ public ClassLoaderInstrumentationModule() {
 
   @Override
   protected String[] additionalHelperClassNames() {
-    return new String[] {"io.opentelemetry.javaagent.tooling.Constants"};
+    return new String[] {
+      "io.opentelemetry.javaagent.tooling.Constants",
+      "io.opentelemetry.javaagent.instrumentation.javaclassloader.ClassLoaderInstrumentation$Holder"
```

Comment:
This class should be picked automaticallty by muzzle, there's no need to specify it.

Reply:
Why `io.opentelemetry.javaagent.tooling.Constants` is needed then?

Reply:
It's not in the `io.opentelemetry.javaagent.instrumentation` or `io.opentelemetry.instrumentation` (except `api`) package, so it does not get detected as a helper class.